src/__tests__/pages/index.test.js

```diff
@@ -0,0 +1,3 @@
+test('Check if reality is correct', () => {
+  expect(2).toBe(1 + 1);
+});
```

Comment:
Nit: Would personally prefer simply `tests` instead of `__tests__`. We don't have `__components__`, `__styles__`, etc either. And I'm not seeing any documentation that says that this is a convention for Next.

Reply:
@simon-debruijn Can you change this and merge the PR after?

Reply:
Remember to also look into the white page of death on Chromatic. I suspect it's actually an issue with the production build of our Storybook (vs the dev mode).